TLSharp.Core/TelegramClient.cs

```diff
@@ -110,15 +112,15 @@ public bool IsUserAuthorized()
 
             var authCheckPhoneRequest = new TLRequestCheckPhone() { phone_number = phoneNumber };
             var completed = false;
-            while(!completed)
+            while (!completed)
```

Comment:
Come on, all these discussions for a space and EOL? I'm sure doing it would be much easier for you than me.